en/docs/learn/user-account-suspension.md

```diff
@@ -118,7 +118,11 @@ task that checks for idle accounts is common to all tenants.
     
     | Field              | Description                                                                                                                                                                                                                                                                                                                                                                                   | Sample Value |
     |--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
-    | Lock Timeout Increment Factor | This specifies the increment factor of the lock timeout. If the lock timeout increment factor is set to 2, then the timeout increments by 2 to the power of the number of failed login attempts. [Timeout increment = (Lock Timeout Increment Factor)<sup>(Number of Failed Attempts)</sup>] (If the number of failed login attempts is 3, timeout increment is 2<sup>3</sup>=8)                                                                                                                                                                                                                                          | 2           |
+    | Account Lock Enabled | This specifies the account locking for failed logins.                                                                                                                                                                                                                                         |            |
+    | Maximum Failed Login Attempts | This specifies the number of failed attempts allows without locking the account.                                                                                                                                                                                                                                      | 5           |
+    | Lock Unlock time | This specifies the account locked time span in minutes.                                                                                                                                                                                                                                          | 5           |
```

Comment:
This specifies how long the account is locked for. The time span is given in minutes.

Reply:
Fixed https://github.com/wso2/docs-is/pull/1409/commits/fcb53cb5030e00b3a00d9054f1bb75f1e310757e